lib/web_ui/dev/integration_tests_manager_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Does this run on CI?

Reply:
No. All CI related stuff is in the TODOs.

Reply:
done.